src/main/java/mate/academy/spring/dto/request/CinemaHallRequestDto.java

```diff
@@ -1,7 +1,14 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Min;
+import javax.validation.constraints.Size;
+import org.springframework.lang.NonNull;
+
 public class CinemaHallRequestDto {
+    @Min(value = 1)
     private int capacity;
+    @NonNull
+    @Size(min = 100)
```

Comment:
what if someone add short description? 
```suggestion
    @Size(min = 10)
```